### PR TITLE
Document parameter for parallel CCM service reconciliation

### PR DESCRIPTION
**Pull request by embik**
This is the documentation companion PR to https://github.com/kubermatic/kubeone/pull/2916, documenting the newly added parameter that can be passed to the CCM addons to configure parallel reconciliation of `Service` objects.

### CODE REVIEW

**Comment by embik** [comment]
/hold

until feature PR is merged.

File: content/kubeone/main/guides/addons/_index.en.md
Comment (xmudrii) [inline content/kubeone/main/guides/addons/_index.en.md:197]: Can we please create a follow up issue in this or kubeone repo to document all well-know parameters? There are in fact many of them and I think it's a great idea to have them documented in a central place like this.

**Review by xmudrii — APPROVED** [review]
Just a tiny follow up.
/lgtm
/approve

File: content/kubeone/main/guides/addons/_index.en.md
Comment (embik) [reply content/kubeone/main/guides/addons/_index.en.md:197]: Created #1518.

Comment (xmudrii) [reply content/kubeone/main/guides/addons/_index.en.md:197]: Thank you!

**Pull request merged**
